### PR TITLE
Update README instructions to use API token template

### PR DESCRIPTION
**Pull request by GregBrimble**
Can be merged once the dashboard change is released.

### CODE REVIEW

**Review by dario-piotrowicz — APPROVED** [review]
awesome 🤩

File: README.md [hunk 1]
```diff
@@ -36,11 +36,10 @@ To activate **_Images_**, please login into your account, select **_Images_** on
 
 ### API token
 
-Before we begin, you also need to create an API token in your Cloudflare account. To do that, [login](https://dash.cloudflare.com/) into your account, and press the **_Create Token_** button under **_My Profile (top right corner) / API Tokens_**.
+[Click here](https://dash.cloudflare.com/profile/api-tokens?permissionGroupKeys=[{"key":"d1","type":"edit"},{"key":"page","type":"edit"},{"key":"images","type":"edit"},{"key":"access_acct","type":"read"},{"key":"workers_kv_storage","type":"edit"},{"key":"dns","type":"edit"},{"key":"access","type":"edit"},{"key":"workers_scripts","type":"edit"}]&name=Wildebeest) to create a Cloudflare API token with the necessary permissions for Wildebesst.
```
Comment (dario-piotrowicz) [inline README.md:39]:
```suggestion
[Click here](https://dash.cloudflare.com/profile/api-tokens?permissionGroupKeys=[{"key":"d1","type":"edit"},{"key":"page","type":"edit"},{"key":"images","type":"edit"},{"key":"access_acct","type":"read"},{"key":"workers_kv_storage","type":"edit"},{"key":"dns","type":"edit"},{"key":"access","type":"edit"},{"key":"workers_scripts","type":"edit"}]&name=Wildebeest) to create a Cloudflare API token with the necessary permissions for Wildebeest.
```

**Comment by GregBrimble** [comment]
@xtuc says he's maybe going to put this elsewhere, so I'll just leave this here unmerged, but ready to go, in case that doesn't work or whatever.

**Comment by xtuc** [comment]
Removing in https://github.com/cloudflare/wildebeest/pull/112

**Review by Saiv46 — APPROVED** [review]
LGTM

**Comment by xtuc** [comment]
We'll use the API token template link in the worker deployement flow.

**Pull request closed without merging**